src/main/java/uk/gov/hmcts/reform/sscs/service/SubmitAppealService.java

```diff
@@ -262,6 +269,20 @@ SscsCaseData convertAppealToSscsCaseData(SyaCaseWrapper appeal) {
         return sscsCaseData;
     }
 
+    private String resolvePostCode(SyaCaseWrapper appeal) {
+        String apellantPostCode = appeal.getAppellant().getContactDetails().getPostCode();
+
+        if (appeal.getIsAppointee()) {
+            return Optional.ofNullable(appeal.getAppointee())
+                .map(SyaAppointee::getContactDetails)
+                .map(SyaContactDetails::getPostCode)
+                .filter(appointeePostCode -> !"".equals(appointeePostCode))
```

Comment:
is it possible for `appointeePostCode` to be `null` here? Or can it only be ""?

Reply:
So this will fall through into returning the apellant post code if the postcode is null or "". Discovered that sometimes ExUI passes an empty field as ""